newspaper/static/index.css

```diff
@@ -78,7 +81,7 @@ form {
     display: flex;
     flex-direction: column;
     margin-left: 6rem;
-    transform: translateY(-50%);
```

Comment:
Removing this makes the input text boxes go wonky. Here is a screenshot of how it looks without `translateY `and with margin instead: 
![image](https://user-images.githubusercontent.com/62475932/199514353-78830f7b-d1c3-4cd8-aeda-6ed5c9d66aaf.png)
Here it is with translate: 
![image](https://user-images.githubusercontent.com/62475932/199514435-ca04a77c-c7ed-45b4-9476-57390e89ed28.png)